dev/CommonStyles/AppBarButton_themeresources.xaml

```diff
@@ -112,17 +119,15 @@
         <Setter Property="Width" Value="68" />
         <Setter Property="UseSystemFocusVisuals" Value="{StaticResource UseSystemFocusVisuals}" />
         <Setter Property="AllowFocusOnInteraction" Value="False" />
+        <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}" />
         <contract6Present:Setter Property="KeyboardAcceleratorPlacementMode" Value="Hidden" />
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="AppBarButton">
                     <Grid x:Name="Root"
+                        Background="Transparent"
                         MinWidth="{TemplateBinding MinWidth}"
-                        MaxWidth="{TemplateBinding MaxWidth}"
-                        Background="{TemplateBinding Background}"
```

Comment:
why do we not need these TemplateBinding's anymore? 

also not sure about the Transparent background (grid is transparent by default)

Reply:
I moved them to the InnerBorder (Background, Border brush/thickness, CornerRadius).

Technically grid has no background by default, the difference being that items without a background set do not get mouse hover events. :/ That's why I had to set it to transparent.

Reply:
right, it's true with hover/press situation.